crates/atiny-checker/src/context.rs

```diff
@@ -52,6 +52,16 @@ impl Default for Ctx {
             TypeSignature::new_opaque("Int".to_string()),
         );
 
+        let int = MonoType::typ("Int".to_string());
+        let sig = MonoType::arrow(int.clone(), MonoType::arrow(int.clone(), int)).to_poly();
+
+        ctx.map.extend([
+            ("add".to_string(), sig.clone()),
```

Comment:
Why not using directly the names like "+", I think it would be easier to read and more beautiful than parsing it to "add"

Reply:
And would provide better error messages